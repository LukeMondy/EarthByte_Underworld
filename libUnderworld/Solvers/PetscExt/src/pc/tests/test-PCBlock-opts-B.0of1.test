EXEC            = ./test-PCBlock-opts.app -pc_block_type upper -pc_block_Q11_pc_type bjacobi -pc_block_Q22_pc_type lu
NPROCS          = 1
TEST_TYPE       = tolerance_diff_pl,1.0e-10
