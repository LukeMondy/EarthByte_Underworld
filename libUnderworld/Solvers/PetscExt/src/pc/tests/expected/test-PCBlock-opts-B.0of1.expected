block_type: upper 
(1,1) pc_type: bjacobi 
(2,2) pc_type: lu 
